.gitignore

```diff
@@ -1,2 +1,3 @@
 /luacov.*
-/site
\ No newline at end of file
+/site
+/lua_install
```

Comment:
Are you running hererocks locally? This is an acceptable change, it's just interesting that this is how you got your Lua install to work 😅

Reply:
This was carry over from the previous pull request. I can remove it if needed.